### ISSUE TITLE
kendoNumericTextBox throwing error on /admin/language/edit for norwegian language (nb-NO)

### ISSUE THREAD

**Issue by lintho** [nopSolutions/nopCommerce#6513]
nopCommerce version: 4.6

Steps to reproduce the problem: 
Edit a norwegian language and open console in chrome.

**Uncaught SyntaxError: Unexpected token '&'**
https://i.imgur.com/676ag7S.png

```
<script type="text/javascript">
    $(document).ready(function() {
        $("#DisplayOrder").kendoNumericTextBox({
            value: "1",
            min: &#x2212;2147483648,  **/* THIS FAILS */**
            max: 2147483647,
            decimals: 0,
            restrictDecimals: true,
            format: "# "
        });
    });
</script>
```

**Comment by lintho** [comment]
Seems to be a problem on all admin pages with numbers

**Comment by exileDev** [comment]
Closed #6513
